airflow/cli/commands/plugins.py

```diff
@@ -0,0 +1,47 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from typing import Dict, List
+
+from airflow import plugins_manager
+from airflow.cli import airflow_cmd, click_output, click_verbose
+from airflow.cli.simple_table import AirflowConsole
+from airflow.plugins_manager import get_plugin_info
+from airflow.utils.cli import suppress_logs_and_warning_click_compatible
+
+
```

Comment:
I removed the 2 unused private functions in `plugins_command.py`.

https://github.com/astronomer/airflow/blob/d39ec4e4590e66ea0ac72dac1e41db0ebb043f81/airflow/cli/commands/plugins_command.py#L26-L36